docs/3.x.x/guides/databases.md

```diff
@@ -0,0 +1,296 @@
+# Databases
+
+Strapi gives you the option to choose the most appropriate database for your project. It currently supports **PostgreSQL**, **MongoDB**, **SQLite**, **MySQL** and
+**MariaDB**. The following documentation covers how to install these databases locally (for development purposes) and on various hosted or cloud server solutions (for staging or production purposes).
+
+(Deploying **Strapi** itself is covered in the [Deployment Guide](/3.x.x/guides/deployment.html).)
+
+**Table of contents:**
+
+- [SQLite Installation](#sqlite-installation)
+- [MongoDB Installation](#mongodb-installation)
+
+
+---
+
+## SQLite Installation
+
+SQLite is the default ([Quick Start](/3.x.x/getting-started/quick-start.html)) and recommended database to quickly deploy an app locally.
```

Comment:
SQLite is the default ([Quick Start](/3.x.x/getting-started/quick-start.html)) and recommended database to quickly create an app locally.

Reply:
done